frontend/src/components/pages/FridgeManagement/FridgeCheckIns/DeleteCheckIns.tsx

```diff
@@ -0,0 +1,119 @@
+import {
+  Button,
+  Container,
+  FormControl,
+  FormErrorMessage,
+  FormHelperText,
+  FormLabel,
+  HStack,
+  Input,
+  Text,
+  useToast,
+} from "@chakra-ui/react";
+import React, { useState } from "react";
+import DatePicker, { DateObject } from "react-multi-date-picker";
+import { useHistory } from "react-router-dom";
+
+import CheckInAPIClient from "../../../../APIClients/CheckInAPIClient";
+import * as Routes from "../../../../constants/Routes";
+import ErrorMessages from "./ErrorMessages";
+
+const DeleteCheckInsPage = () => {
+  const toast = useToast();
+  const history = useHistory();
+  const [dateRange, setDateRange] = useState<DateObject[]>([
+    new DateObject(),
+    new DateObject().add(1, "days"),
+  ]);
+  const [formErrors, setFormErrors] = useState({
+    dateRange: "",
+  });
+
+  const validateForm = () => {
+    const newErrors = {
+      dateRange: "",
+    };
+    let valid = true;
+    if (!dateRange[0] || !dateRange[1]) {
+      valid = false;
+      newErrors.dateRange = ErrorMessages.bothDateFieldsRequired;
+    }
+    setFormErrors(newErrors);
+    return valid;
+  };
+
+  const onSaveClick = async () => {
+    const isValid = validateForm();
+    if (!isValid) {
+      return;
+    }
+    const res = await CheckInAPIClient.deleteCheckInsByDateRange(
+      dateRange[0].toString(),
+      dateRange[1].add(1, "days").toString(),
```

Comment:
end date wasn't being deleted, but it seems to work with this little hack - not sure if the issue is with `deleteCheckInsByDateRange` or smth else